ISAAR.MSolve.SamplesConsole/Examples/Static/Linear/Hexa8SimpleExample.cs

```diff
@@ -0,0 +1,140 @@
+            return nodes;
+        }
+
+        static void Main(string[] args)
+        {
+            VectorExtensions.AssignTotalAffinityCount();
+            double youngModulus = 200.0e06;
+            double poissonRatio = 0.3;
+            double nodalLoad = 25.0;
+
+            // Create a new elastic 3D material
+            ElasticMaterial3D material = new ElasticMaterial3D()
+            {
+                YoungModulus = youngModulus,
+                PoissonRatio = poissonRatio,
+            };
+
+            // Node creation
+            IList<Node> nodes = CreateNodes();
+
+            // Model creation
+            Model model = new Model();
+
+            // Add a single subdomain to the model
+            model.SubdomainsDictionary.Add(1, new Subdomain() { ID = 1 });
+
+            // Add nodes to the nodes dictonary of the model
+            for(int i=0; i < nodes.Count; ++i)
+            {
+                model.NodesDictionary.Add(i + 1, nodes[i]);
+            }
+
+            // Constraint bottom nodes of the model
+            model.NodesDictionary[1].Constraints.Add(DOFType.X);
+            model.NodesDictionary[1].Constraints.Add(DOFType.Y);
+            model.NodesDictionary[1].Constraints.Add(DOFType.Z);
+            model.NodesDictionary[2].Constraints.Add(DOFType.X);
+            model.NodesDictionary[2].Constraints.Add(DOFType.Y);
+            model.NodesDictionary[2].Constraints.Add(DOFType.Z);
+            model.NodesDictionary[3].Constraints.Add(DOFType.X);
+            model.NodesDictionary[3].Constraints.Add(DOFType.Y);
+            model.NodesDictionary[3].Constraints.Add(DOFType.Z);
+            model.NodesDictionary[4].Constraints.Add(DOFType.X);
+            model.NodesDictionary[4].Constraints.Add(DOFType.Y);
+            model.NodesDictionary[4].Constraints.Add(DOFType.Z);
+            
+
+            // Create a new Hexa8 element
+            var element = new Element()
+            {
+                ID = 1,
+                ElementType = new Hexa8(material)
+            };
+
+            // Add nodes to the created element
+            element.AddNode(model.NodesDictionary[1]);
+            element.AddNode(model.NodesDictionary[2]);
+            element.AddNode(model.NodesDictionary[3]);
+            element.AddNode(model.NodesDictionary[4]);
+            element.AddNode(model.NodesDictionary[5]);
+            element.AddNode(model.NodesDictionary[6]);
+            element.AddNode(model.NodesDictionary[7]);
+            element.AddNode(model.NodesDictionary[8]);
+
+            // Add Hexa element to the element and subdomains dictionary of the model
+            model.ElementsDictionary.Add(element.ID, element);
+            model.SubdomainsDictionary[1].ElementsDictionary.Add(element.ID, element);
+
+            // Add nodal load values at the top nodes of the model
+            model.Loads.Add(new Load() { Amount = -nodalLoad, Node = model.NodesDictionary[5], DOF = DOFType.Z });
```

Comment:
Remove solution file from your pull request since it does not reflect any actual code changes.